expense-tracker/src/db/procedures/fetch_wallet_details_v1dot0.sql

```diff
@@ -0,0 +1,10 @@
+DELIMITER $$
+create procedure fetch_user_details_v1dot0 (
```

Comment:
Rename the procedure to fetch wallet details. If we run this as is this will replace the actual fetch user detail procedure and u we will be wondering !!